README.md

```diff
@@ -12,44 +12,6 @@ the Getting Started section below. Please feel free to [file
 issues](https://github.com/Microsoft/cadl/issues) for any issues you encounter while
 using the preview.
 
```

Comment:
Are you going to add the "reason to use cadl" paragraph I suggested? It should go right after the first paragraph above (line 9).

Reply:
Added your paragraph. Will add the more specific bullet items to Azure one as it is more specific. (OpenAPI)